docs/spec/system.json

```diff
@@ -17,6 +17,48 @@
             "description": "Name of the system platform the agent is running on.",
             "type": ["string", "null"],
             "maxLength": 1024
+        },
+        "container": {
+            "properties": {
+                "id" : {
+                    "description": "Container ID",
+                    "type": ["string"],
+                    "maxLength": 1024
+                }
+            },
+            "required": ["id"]
```

Comment:
this is required so as to not have an empty `container`? 
if so, probably i am overthinking this, but it could happen that an `id` is not possible to retrieve in some case, then at some point we add another field under `container` like `name`, but agents can't send  `container.name` without a `container.id`?